components/TestimoniesOnHomePage/TestimoniesOnHomePage.js

```diff
@@ -2,15 +2,22 @@ import React from "react"
 import { useRecentTestimony } from "../db"
 import { Button, Container, Row, Spinner, Table } from "react-bootstrap"
 import { useRouter } from "next/router"
+import { formatBillId } from "../formatting"
 
 // the word "testimonies": In more general, commonly used, contexts, the plural form will also be testimony.  However, in more specific contexts, the plural form can also be testimonies e.g. in reference to various types of testimonies or a collection of testimonies.
 
 const TestimonyRow = ({ testimony }) => {
   const router = useRouter()
   return (
     <tr>
-      <td>{testimony.billId}</td>
-      <td>{testimony.position}</td>
+      <td>
+        <Button
```

Comment:
Let's optimize these buttons a bit by specifying the url they link to rather than pushing the route in the click handler. The buttons will render as links and will allow them to preload the page when you hover over them. See the [buttons on the home page](https://github.com/codeforboston/advocacy-maps/blob/2b10afa5e9be872dd5799c17a9b9d88b1ba68d94/pages/index.tsx#L32). 

```
          <Wrap href={`/bill?id=${testimony.billId}`}>
            <Button>{formatBillId(testimony.billId)}</Button>
          </Wrap>
```

Reply:
Is this something that should be done on the Bills with Upcoming Hearings as well?

Reply:
Yeah @almaraz333 , it would be great if we could update that table and the author column on this one!